### PR TITLE
Tinovi vendor

### PR DESCRIPTION
**Pull request by datatoi**
#### Summary

...

#### Changes
- added tinovi vendor

### CODE REVIEW

**Comment by Jaime-Trinidad** [comment]
Hi @datatoi thanks for adding devices, please sign CLA.

File: vendor/tinovi/pm-io-5-sm-codec.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: pm-io-5-sm.js
+
```
Comment (Jaime-Trinidad) [inline vendor/tinovi/pm-io-5-sm-codec.yaml:5]: is it possible to you to add an example?

**Comment by Jaime-Trinidad** [comment]
@datatoi please run `make validate fmt`

**Comment by datatoi** [comment]
Will create new PR

**Pull request closed without merging**